index.html

```diff
@@ -49,5 +49,167 @@ <h1><span class="hero__title-dark">Hey there. I'm Daniel Selase</span><span clas
+                        <div class="card__img"></div>
+                        <h4 class="card__title">Multi-Post Stories Gain+Glory</h4>
+                        <ul class="card__tags-works">
+                            <li class="card__tags-bg"><a href="#">Ruby on rails</a></li>
+                            <li class="card__tags-bg"><a href="#">css</a></li>
+                            <li class="card__tags-bg"><a href="#">JavaScript</a></li>
+                            <li class="card__tags-bg"><a href="#">html</a></li>
+                        </ul>
+                        <button class="btn">See Project</button>
+                </div>
+                <div class="card">
+                    <div class="card__img"></div>
+                    <h4 class="card__title">Multi-Post Stories Gain+Glory</h4>
+                    <ul class="card__tags-works">
+                        <li class="card__tags-bg"><a href="#">Ruby on rails</a></li>
+                        <li class="card__tags-bg"><a href="#">css</a></li>
+                        <li class="card__tags-bg"><a href="#">JavaScript</a></li>
+                        <li class="card__tags-bg"><a href="#">html</a></li>
+                    </ul>
+                    <button class="btn">See Project</button>
+            </div>
+            <div class="card">
+                <div class="card__img"></div>
+                <h4 class="card__title">Multi-Post Stories Gain+Glory</h4>
+                <ul class="card__tags-works">
+                    <li class="card__tags-bg"><a href="#">Ruby on rails</a></li>
+                    <li class="card__tags-bg"><a href="#">css</a></li>
+                    <li class="card__tags-bg"><a href="#">JavaScript</a></li>
+                    <li class="card__tags-bg"><a href="#">html</a></li>
+                </ul>
+                <button class="btn">See Project</button>
+        </div>
+        <div class="card">
+            <div class="card__img"></div>
+            <h4 class="card__title">Multi-Post Stories Gain+Glory</h4>
+            <ul class="card__tags-works">
+                <li class="card__tags-bg"><a href="#">Ruby on rails</a></li>
+                <li class="card__tags-bg"><a href="#">css</a></li>
+                <li class="card__tags-bg"><a href="#">JavaScript</a></li>
+                <li class="card__tags-bg"><a href="#">html</a></li>
+            </ul>
+            <button class="btn">See Project</button>
+    </div>
+    <div class="card">
+        <div class="card__img"></div>
+        <h4 class="card__title">Multi-Post Stories Gain+Glory</h4>
+        <ul class="card__tags-works">
+            <li class="card__tags-bg"><a href="#">Ruby on rails</a></li>
+            <li class="card__tags-bg"><a href="#">css</a></li>
+            <li class="card__tags-bg"><a href="#">JavaScript</a></li>
+            <li class="card__tags-bg"><a href="#">html</a></li>
+        </ul>
+        <button class="btn">See Project</button>
+</div>
```

Comment:
:red_circle:  This is the element `<div class="card">` that should be implemented using the flexbox as stated in The [project requirement ](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md). The Flexbox should be applied to the cards containers. Please read the second line in the screenshot :point_down: 

![requirement](https://user-images.githubusercontent.com/26736582/200043935-074b671b-0b02-48b3-90f4-4e73a7124d9d.png)

Please  make sure you follow these requirements because they are parts of the project implementation